packages/harpguru-core/src/components/harp-row/harp-row.test.tsx

```diff
@@ -11,11 +11,24 @@ import { HarpRow } from './harp-row'
 
 jest.mock('reactn')
 const mockUseGlobal = useGlobal as jest.Mock
+// TODO: Should be shared resource
```

Comment:
Already addressed in a code review above, but this comment needs removing and the code modifying here too.

Reply:
Done